xformers/components/attention/csrc/cuda/mem_eff_attention/kernel_backward.h

```diff
@@ -626,6 +641,7 @@ struct AttentionBackwardKernel {
           problem_size.k());
 
       if (!kOutputInRF) {
+        createEpilogueIter().prefetch_all();
```

Comment:
You don't store the return value of this function and instead recreate it just below to save some registers, which makes things faster according to Daniel's benchmarks